src/PromiseRouter.js

```diff
@@ -186,11 +207,11 @@ function makeExpressHandler(promiseHandler) {
         }
         res.json(result.response);
       }, (e) => {
-        log.verbose('error:', e);
+        log.error(`Error generating response. ${e}`, {error: e});
```

Comment:
This is also not correct and will just just log `Error generating response. [object Object]`